### DISCUSSION TITLE
Where should the WordPress Brand Writing Style Guide live?

### DISCUSSION THREAD

**Discussion by sereedmedia** [WordPress/Marketing-Team#305] [General]
Currently the WordPress Brand Writing Style Guide in housed in the Make WordPress Marketing Team Handbook.

This doesn't have a clear way to track updates or changes, nor a way to notify other teams, such as Docs, Polyglots, WPTV, or Core.

Where could the WordPress Brand Style Guide live or where should the main repository for the content be where it could be more prominent and accessible?

**Comment by eidolonnight** [comment]
Is this in reference to the [**WordPress Brand Writing Style Guide**](https://make.wordpress.org/marketing/handbook/resources/style-guide-and-brand-book/)? As far as I am am aware, we do not yet have a general brand guide.

**Comment by sereedmedia** [comment]
Yes. It's been called many things, at one point it was the [Style Guide and Brand Book](https://web.archive.org/web/20200928162451/https://make.wordpress.org/marketing/handbook/resources/style-guide-and-brand-book/). You can see this in the slug for what is currently called the "WordPress Brand Writing Style Guide", which is "style-guide-and-brand-book/". 

For clarity, I have updated the ticket from "WordPress Brand Style Guide" to "WordPress Brand Writing Style Guide".

This very side note is a great illustration of why we need a place to manage revision tracking. Until this is in Core, I think GitHub should be the place for it. Because we did at one point have at least the beginnings of a Style Guide and Brand Book, and now we do not, which is not quite the same thing as "not yet" having one.

**Comment by eidolonnight** [comment]
Cool! Since the Design team owns the visual brand, I think it makes sense for this guide to focus on brand _writing_. That said, I believe the current guide primarily applies to marketing deliverables and does not cover UX, docs, etc. Personally, I think these areas will likely require their own guides, and there should be collaboration between the teams (rather than one giant guide that tries to cover everything).

As for where it's kept, I really like how https://make.wordpress.org/docs/style-guide/ is managed. It appears in the handbook, but is edited via Github (I.e. benefits from relatively easy-to-view version control).

**Comment by eidolonnight** [comment]
- [Human WordPress Brand Writing Style Guide](https://make.wordpress.org/marketing/handbook/resources/style-guide-and-brand-book/)
- [AI Agent WordPress Brand Writing Style Guide](https://github.com/WordPress/marketing/blob/main/shared/references/wordpress-brand-writing-style-guide.md)
